apps/snapodds/src/config/application-config.ts

```diff
@@ -1,5 +1,6 @@
 import { AccessToken, TvSearchResultEntry } from '@response/typings';
 import { LogLevel } from '../services/logger/log-level';
+import { SdkMode } from './sdk-mode';
```

Comment:
I think that it will be better to move all classes from this package to services/config.

Reply:
done